src/cli/error.rs

```diff
@@ -67,10 +67,18 @@ impl CLIError {
     }
 }
 
+const ERROR_PREFIX: &str = "[ERROR] ";
+
 fn margin(str: &str, margin: usize) -> String {
     let mut result = String::new();
     for line in str.split_inclusive('\n') {
-        result.push_str(&format!("{}{}", " ".repeat(margin), line));
+        let index = line
+            .find(ERROR_PREFIX)
+            .map(|i| i + ERROR_PREFIX.len())
+            .unwrap_or(0);
+        let (prefix, line) = line.split_at(index);
+
+        result.push_str(&format!("{}{}{}", prefix, " ".repeat(margin), line));
```

Comment:
This is necessary as the margin was being added to the description before the prefix for the description.